src/polyswarm_api/resources.py

```diff
@@ -984,10 +984,19 @@ def sandbox(cls, api, sha256):
         return core.PolyswarmRequest(
             api,
             {
-                'method':
-                'POST',
-                'url':
-                '{}/consumer/submission/{}/sandbox/{}'.format(api.uri, api.community, sha256)
+                'method': 'POST',
+                'url': '{}/consumer/submission/{}/sandbox/{}'.format(api.uri, api.community, sha256)
+            },
+            result_parser=cls,
+        ).execute()
+
+    @classmethod
+    def sandbox_list(cls, api):
```

Comment:
Please use the default routes there, check the live hunts on how to do that, we have listing methods there. We should try to use the default stuff as much as possible.

Reply:
We can jump in a call if you think that will help.

Reply:
Sure, makes sense. I made the change but had to override `_endpoint` to format the uri with the community: https://github.com/polyswarm/polyswarm-api/pull/190/files#diff-5b575475ce5b098eb1a647726381758cb9b5f86dc914ce3eab3a7db8d25dc3b3R995

Not sure if there is a better way to do that but this seemed the most simple.